src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -876,23 +866,6 @@ tr.status-warning td {
     color: var(--color-error);
 }
 
-{# Syntax highlighting
```

Comment:
shouldn't syntax highlighting styles be kept in global styles ? Or is there only one panel that relies on syntax highlighting ?

Reply:
I tested all my panels and I saw no changes after removing this. I looked for in the source code and templates and I can't find references to this. My guess is that it's some unused legacy styles.